action.yml

```diff
@@ -465,7 +478,7 @@ runs:
         echo "Finished validation using Java validator (Conformance resources) ..."
       shell: bash
       env:
-       FHIR_VERSION: "4.0"
+       FHIR_VERSION: ${{ env.FHIR_VERSION }}
```

Comment:
I don't think this works, you need to transform "4.0.1" -> "4.0", etc.

Reply:
Why? 4.0.1 is also a valid option to use? https://confluence.hl7.org/spaces/FHIR/pages/35718580/Using+the+FHIR+Validator#UsingtheFHIRValidator-Choosingtheversion

Reply:
Why do you then need the detectedFhirVersion var and not re-use $fhirVersion?